client/src/modules/CrossCheckPairs/data/getCrossCheckPairsColumns.tsx

```diff
@@ -0,0 +1,117 @@
+import { CrossCheckPairDto } from 'api';
+import { ColumnType } from 'antd/lib/table/interface';
+import { omit } from 'lodash';
+import { dateTimeRenderer, getColumnSearchProps } from 'components/Table';
+import { GithubAvatar } from 'components/GithubAvatar';
+import { Button } from 'antd';
+
+export const fields = {
+  task: 'task',
+  checker: 'checker',
+  student: 'student',
+  url: 'url',
+  score: 'score',
+  submittedDate: 'submittedDate',
+  reviewedDate: 'reviewedDate',
+};
+
+export interface CustomColumnType<RecordType> extends ColumnType<RecordType> {
+  sorterField?: string;
+}
+
+const renderGithubLink = (value: string) => (
+  <div>
+    {value ? (
```

Comment:
if value is not defined no need to return div

Reply:
fixed